proto-idls/src/main/java/com/craftsmanship/tfm/testing/persistence/ItemPersistenceStub.java

```diff
@@ -28,6 +28,10 @@ public Item get(Long id) {
     }
 
     public Item update(Long id, Item item) {
+        if (items.get(id) == null) {
+            return null;
```

Comment:
if this happen, maybe we should log that there was an attempt to update a non existing item.